hack/boilerplate.go.txt

```diff
@@ -1,17 +1,13 @@
-// Copyright 2018, OpenCensus Authors
+// Copyright YEAR Red Hat, Inc. and/or its affiliates
```

Comment:
year should be replaced here

Reply:
It's replaced by the openapi tool. This is a template.